.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <Objective-C-extensions>
```

Comment:
Wait, wut?

Reply:
I wonder if it pulled this from my AppCode config... I don't feel like we need to include this here....

Reply:
I looked at my existing AS config, and it has this element.  It might be done (over) re-use on the name

Reply:
ok, leaving it.

I am unable to delete it without AS putting it back so we have no choice.